extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -117,6 +122,36 @@ export class AmpAdUIHandler {
   }
 
   /**
+   * See BaseElement method.
+   */
+  createPlaceholderCallback() {
+    if (!isExperimentOn(this.baseInstance_.win, UX_EXPERIMENT)) {
+      return null;
+    }
+    const placeholder = createElementWithAttributes(this.doc_, 'div', {
```

Comment:
i think we can move this into the createDefaultHolder_ method. in fact, we can merge this whole method into it.